<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="meta_placeholder" native="true"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="create_placeholders_panel">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_side_2">
       <item>
        <widget class="QLabel" name="placeholder_label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Placeholder Insertions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="placeholder_scroll_area">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="placeholder_container">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>228</width>
            <height>413</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="_3">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_placeholder_button">
         <property name="text">
          <string>Add Placeholder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="templated_content" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="templated_meta_placeholder" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="RootTreeView" name="tree_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="LivePVTableView" name="sub_pv_table_view"/>
          <widget class="NestableTableView" name="sub_coll_table_view"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="swap_mode_button">
         <property name="text">
          <string>&lt; &gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save Template</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="substitute_panel">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_side">
       <item>
        <widget class="QLabel" name="substitute_label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Template Substitutions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="substitute_scroll_area">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="substitute_container">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>228</width>
            <height>413</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="_2">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="create_coll_button">
         <property name="text">
          <string>Create Collection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LivePVTableView</class>
   <extends>QTableView</extends>
   <header>superscore.widgets.views</header>
  </customwidget>
  <customwidget>
   <class>NestableTableView</class>
   <extends>QTableView</extends>
   <header>superscore.widgets.views</header>
  </customwidget>
  <customwidget>
   <class>RootTreeView</class>
   <extends>QTreeView</extends>
   <header>superscore.widgets.views</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
